images/linux/scripts/installers/1804/powershellcore.sh

```diff
@@ -8,11 +8,15 @@
 source $HELPER_SCRIPTS/document.sh
 
 LSB_RELEASE=$(lsb_release -rs)
+echo "Addin the repository [microsoft-ubuntu-bionic-prod] to /etc/apt/sources.list.d/dotnetdev.list"
+sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/repos/microsoft-ubuntu-bionic-prod bionic main" > /etc/apt/sources.list.d/dotnetdev.list'
 
 # Install Powershell
+echo "Running the PowerShell core installation"
 apt-get install -y powershell
 
-# Temp fix based on: https://github.com/PowerShell/PowerShell/issues/9746 
+# Temp fix based on: https://github.com/PowerShell/PowerShell/issues/9746
+echo "Remove the libicu64"
```

Comment:
Do we really need this echo? I mean, apt remove is pretty self explanatory itself, at least if you don't suppress it's output, and current echo doesn't tell anything specific except that libicu64 gonna be removed.

Reply:
I've already changed this line here https://github.com/actions/virtual-environments/pull/453, so there will be a conflict

Reply:
excess 'echo's have been removed: https://github.com/andy-mishechkin/virtual-environments/commit/cdac96cfe3d2a504008b89e7b8c450dcdaa6f3d3